docs/install/index.rst

```diff
@@ -37,17 +41,54 @@ To check your Gammapy installation, you can use this command:
 
     gammapy info
 
-Congratulations! You are all set to start using Gammapy!
 
-If you're new to Python, ipython and Jupyter, read the :ref:`getting-started`
-guide. To learn how to use Gammapy, go to :ref:`tutorials`.
+Using other package managers
+----------------------------
 
 The following pages contain detailed information about Gammapy dependencies in
 various installation options:
 
-.. toctree::
-    :maxdepth: 1
+* See gammapy install-dependencies :ref:`install-dependencies`
+* If you want to install gammapy with pip see :ref:`install-pip`
+* If you want ot use other package manager see :ref:`install-other`
+
+
+Download tutorials
+------------------
+
+You can now proceed to download the Gammapy tutorial notebooks and the example
+datasets used there (at the moment from CTA, H.E.S.S. and Fermi-LAT). The total
+size to download is about 100 MB.
+
+.. code-block:: bash
+
+    gammapy download tutorials --release 0.17
+    cd gammapy-tutorials
+    export GAMMAPY_DATA=$PWD/datasets
+
+You might want to put the definition of the ``$GAMMAPY_DATA`` environment
+variable in your shell profile setup file that is executed when you open a new
+terminal (for example ``$HOME/.bash_profile``).
```

Comment:
I think in linux machines it's `$HOME/.bashrc` ?

Reply:
Technically, when you open a new terminal, it is probably `$HOME/.bashrc` that is executed.